src/altair/helpers.rs

```diff
@@ -69,9 +103,28 @@ pub fn get_next_sync_committee<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
-) -> Result<()> {
-    Ok(())
+    context: &Context,
+) -> Result<SyncCommittee<SYNC_COMMITTEE_SIZE>> {
+    // Return the next sync committee, with possible pubkey duplicates.
+    let indices = get_next_sync_committee_indices(state, context)?;
+    let public_keys = state
+        .validators
+        .iter()
+        .enumerate()
+        .filter_map(|(i, v)| {
+            if indices.contains(&i) {
```

Comment:
also, if we are only ever using the `indices` as a set to check membership (I believe we are) then `get_next_sync_committee_indices` should collect into a `HashSet`